subprojects/plugins/src/main/java/org/gradle/api/plugins/JavaPluginExtension.java

```diff
@@ -75,7 +75,12 @@
     void disableAutoTargetJvm();
 
     /**
-     * Automatically package Javadoc during publishing and publish a variant 'javadocElements' with an artifact '-javadoc.jar'.
+     * Adds a task {@code javadocJar} that will package the output of the {@code javadoc} task in a JAR with classifier {@code javadoc}.
+     * <P>
+     * The produced artifact is registered as a documentation variant on the {@code java} component and added as a dependency on the {@code assemble} task.
+     * This means that if the {@code maven-publish} or {@code ivy-publish} is also applied, the javadoc JAR will be published.
```

Comment:
:o: remove "the"

or change it to

the maven-publish or ivy-publish **plugin**